content/api/templates.apib

```diff
@@ -556,26 +402,136 @@ If the template delete API call succeeds, then ALL versions of the template will
 
         {}
 
-+ Response 404 (application/json)
++ Response 409 (application/json)
 
         {
             "errors": [
                 {
-                    "message": "resource not found",
-                    "code": "1600",
-                    "description": "Template does not exist"
+                    "message": "resource conflict",
+                    "code": "1602",
+                    "description": "Template is in use by msg generation"
                 }
             ]
         }
 
-+ Response 409 (application/json)
+### List all Templates [GET /templates{?draft,shared_with_subaccounts}]
+
++ Parameters
+    + draft (optional, boolean, `true`) ...If true, returns the most recent draft template.  If false, returns the most recent published template. When not provided, returns the most recent template version regardless of draft or published.
+    + shared_with_subaccounts (optional, boolean, `true`) ...If true, returns only shared templates. If false, returns only non-shared templates. When not provided, returns both shared and non-shared templates.
+
+
+Returns an array of most recent version of each template in your account.
+Each object returned is an abbreviated version of the full template object with
+the following attributes:
+
+
++ Request
+
+    + Headers
+
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+            Accept: application/json
+
++ Response 200 (application/json)
 
         {
-            "errors": [
+            "results": [
                 {
-                    "message": "resource conflict",
-                    "code": "1602",
-                    "description": "Template is in use by msg generation"
+                    "id": "summer_sale",
+                    "name": "Summer Sale!",
+                    "published": true,
+                    "description": "",
+                    "has_draft": true,
+                    "has_published": true,
+                    "last_update_time": "2017-08-11T12:12:12+00:00",
+                    "shared_with_subaccount": true
+                },
+                {
+                    "id": "daily",
+                    "name": "daily",
+                    "published": false,
+                    "description": "Daily roundup email.",
+                    "has_draft": true,
+                    "has_published": true,
+                    "last_use": "2018-05-08T14:49:03+00:00"
+                    "last_update_time": "2018-02-10T14:15:16+00:00",
+                    "shared_with_subaccount": false
                 }
             ]
         }
+
+
+
+### Error Object
```

Comment:
Everything below this can not be here. Since it is after the request but before the next resource ("Create an RFC 822 Template") it has no parent and so won't be rendered correctly. 

I suggest moving it to the top, after the  "Template Object"